docs/userguides/console.md

```diff
@@ -0,0 +1,62 @@
+# Ape Console
+
+Ape provides an [IPython](https://ipython.readthedocs.io/) interactive console with useful pre-defined locals to interact with your project.
+
+```bash
+ape console --network ethereum:mainnet
+
+In [1]: networks.active_provider.get_block("latest")
```

Comment:
Can you use a different call here? The structure of the block is something we are working on, it might change and we don't want to have to update the block struct here

Reply:
Do you have a recommendation?  I'm still familiarizing myself with the Ape API.

Reply:
```py
chain.blocks.head.timestamp
```